### PR TITLE
Update config.php

### PR DESCRIPTION
**Pull request by zandiarash**
برای درست کردن منوی کشویی بغل سمت راست ترتیب ul و li اشتباه بود که اصلاح کردم
با تشکر از همگی دوستان.

### CODE REVIEW

**Comment by MSDehghan** [comment]
خیلی ممنون از اینکه انجام دادید اما آی دی ها و کلاس هایی مثل 
apadana-block-posts-categories
 توی فایل
 engine.css
 تعریف شده اند و اگه تغیر داده بشن کل قالب ها ی نوشته شده به هم میریزه. اگه میشه این ها رو درست کنید.
بازم خیلی خیلی ممنون

**Comment by zandiarash** [comment]
سلام
ببینید من خیلی مبتدی هستم تو طراحی وب عذر میخوام اگه تغییرم مفید نبود.
راستی این کد ها رو هم توی body.tpl تم سایت گزاشتم.
اما هیچ چیز به هم نریخت که.
منوی سایتمونم الان کشوییه.
با این حال حرف شما صحیحه و ممکنه سایت به ههم بریزه.



<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>

   ( function( $ ) {
$( document ).ready(function() {
$('#accordion > li').mouseenter(function() {
  $('#accordion li').removeClass('active');
  $(this).closest('li').addClass('active');
 var checkElement = $(this).next();
  if((checkElement.is('ul')) && (checkElement.is(':visible'))) {
 $(this).closest('li').removeClass('active');
    checkElement.slideUp('normal');
  }
  if((checkElement.is('ul')) && (!checkElement.is(':visible'))) {
    $('#accordion ul ul:visible').slideUp('normal');
    checkElement.slideDown('normal');
  }
  if($(this).closest('li').find('ul').children().length == 0) {
    return true;
  } else {
    return false;
  }
});
});
} )( jQuery );

<STYLE>
#accordion {
    list-style: none;
    padding: 0 0 0 0;
    width: 170px;
}
#accordion li{
    display: block;
    background-color: #dbeeff;
    font-weight: bold;
    margin: 1px;
    cursor: pointer;
    padding: 5 5 5 7px;
    list-style: circle;
    -moz-border-radius: 10px;
    -webkit-border-radius: 10px;
    border-radius: 8px;
}
#accordion ul {
    list-style: none;
    padding: 0 0 0 0;
    display: none;
}
#accordion ul li{
    font-weight: normal;
    cursor: auto;
    background-color: #b5e6ff;
    padding: 0 0 0 7px;
}
#accordion a {
    text-decoration: none;
}
#accordion a:hover {
    text-decoration: underline;
}
</STYLE>
